mage_ai/data_preparation/templates/data_exporters/default.jinja

```diff
@@ -1,16 +1,14 @@
-from pandas import DataFrame
-
 if 'data_exporter' not in globals():
     from mage_ai.data_preparation.decorators import data_exporter
 
 
 @data_exporter
-def export_data(df: DataFrame, **kwargs):
+def export_data(*args, **kwargs):
     """
     Exports data to some source
 
     Args:
-        df (DataFrame): Data frame to export to
+        df (optional): Data frame to export to
```

Comment:
`df` is not in the args anymore. So you can write
```
args: The input variables from upstream blocks
```

Reply:
Okay, I had left the `df` in there as an "optional" arg so users would know they can still pass in data frames, but that should be implied.